sqs/src/main/scala/akka/stream/alpakka/sqs/SqsModel.scala

```diff
@@ -35,14 +36,14 @@ object MessageAction {
   }
 
   final object Delete {
-    def apply(message: Message): MessageAction = new Delete(message)
+    def apply(message: Message): Delete = new Delete(message)
```

Comment:
I'm looking into the MiMa differences and this change shows up. It should stay as `MessageAction` which makes the generics a bit easier for Java users.

Reply:
The Java users should use `MessageAction.delete(message: Message): MessageAction` instead `apply`.

Reply:
Of course. Thank you.